src/server/index.js

```diff
@@ -109,47 +105,39 @@ function graphQLResponse (filter, page, response) {
       }
     }`
 
-  const payload = {
-    query
-  }
+  const payload = JSON.stringify({ query })
 
-  axios
-    .post(API_URL, payload)
-    .then((result) => {
-      response.send(result.data.data.hn[queryType])
+  try {
+    const res = await fetch(API_URL, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: payload
     })
-    .catch(error => {
-      error.statusCode = error.statusCode || 500
+    const result = await res.json()
```

Comment:
We need to re-add the error handling - axios always threw an error if the status code was > 200. Where as the fetch api doesn't.

Same idea from my previous comment:

```js
const response = await fetch(url)

if (response.ok) {
  // all good
} else {
 // handle error - use existing logic
}
```

Reply:
:ok_hand: that's right, though ideally we should check for emptiness of the response or invalid content too, a reverse proxy can for example provide a 200 with an html instead of the json and it should be an error

Reply:
You're right - we're doing the bare minimum here and getting away with it 😅